src/utils.ts

```diff
@@ -27,3 +27,42 @@ export async function wait(seconds = 1) {
     }, seconds * 1000)
   })
 }
+
```

Comment:
Shorter implementation for `wait()`:

```
export async function wait(seconds = 1) {
  return new Promise((resolve) => setTimeout(resolve, seconds * 1000))
}
```

When `utils.ts` only consists of one function which is only used in `src/cache-worker.ts` at the end, I would move the function there as well...